views/data-analysis/error.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+//******************************************************************************
```

Comment:
Why don't you use the /site/error page ? 

(in your controller : 

```php
return $this->render('/site/error', [
                    'name' => Yii::t('app/messages','Internal error'),
                    'message' => Yii::t('app/messages', 'An error occurred.')]);
```
)

Reply:
Done